### PR TITLE
On-chain deployment cleanup

### PR DESCRIPTION
**Pull request by yvonneanne**
Following advice from #docs-working-group

### CODE REVIEW

File: modules/quickstart/pages/network-quickstart.adoc [hunk 1]
```diff
@@ -10,12 +10,12 @@
 ifdef::env-github,env-browser[:outfilesuffix:.adoc]
 
 [[net-quick-start]]
-This _Quick Start_ scenario assumes that you are installing the {sdk-short-name} for the first time and deploying the default project on the {IC} **running remotely**.
+This _Quick Start_ scenario assumes that you are installing the {sdk-short-name} for the first time and deploying the default project on the {IC} blockchain.
```
Comment (andreacerulli) [inline modules/quickstart/pages/network-quickstart.adoc:13]:
```suggestion
This _Quick Start_ scenario assumes that you are installing the {sdk-short-name} for the first time and deploying the default project on the {IC} blockchain mainnet.
```

File: modules/quickstart/pages/network-quickstart.adoc [hunk 2]
```diff
@@ -122,12 +122,12 @@ cd hello
 ----
 
 [[ping-the-network]]
-== Check the connection to the network
+== Check the connection to the {IC} blockchain
```
Comment (andreacerulli) [inline modules/quickstart/pages/network-quickstart.adoc:125]: How about:
```suggestion
== Check the connection to the {IC} mainnet
```

File: modules/quickstart/pages/network-quickstart.adoc [hunk 3]
```diff
@@ -122,12 +122,12 @@ cd hello
...
 
-Before you can deploy your first project on the {IC}, you need to connect to the network gateway running remotely on a subnet that you can access.
-There is a reserved network alias that you can use to access the {IC} running remotely on nodes in data centers. The network alias is a system setting that's defined internally, so there's nothing you need to configure in your projects by default.
+Before you can deploy your first project on the {IC} blockchain, you need to connect to the network gateway running remotely on a subnet that you can access.
```
Comment (andreacerulli) [inline modules/quickstart/pages/network-quickstart.adoc:127]: I believe we can remove this. During the hackathon I had to do nothing of this sort. 
```suggestion

```

File: modules/quickstart/pages/network-quickstart.adoc [hunk 4]
```diff
@@ -122,12 +122,12 @@ cd hello
...
+There is a reserved network alias that you can use to access the {IC} blockchain. The network alias is a system setting that's defined internally, so there's nothing you need to configure in your projects by default.
```
Comment (andreacerulli) [inline modules/quickstart/pages/network-quickstart.adoc:128]:
```suggestion
There is a reserved network alias that you can use to access the {IC} blockchain mainnet. The network alias is a system setting that's defined internally, so there's nothing you need to configure in your projects by default.
```

File: modules/quickstart/pages/network-quickstart.adoc [hunk 5]
```diff
@@ -122,12 +122,12 @@ cd hello
...
 
-To check your connection to the {IC} network:
+To check your connection to the {IC} blockchain:
 
 [arabic]
 . Check that you are in the root directory for your project, if needed.
```
Comment (andreacerulli) [inline modules/quickstart/pages/network-quickstart.adoc:133]: Maybe in the line below (I cannot comment) we can change "network" with "mainnet". But to me this is also fine

File: modules/quickstart/pages/network-quickstart.adoc [hunk 6]
```diff
@@ -343,7 +343,7 @@ For more information about the commands and methods available for working with t
 
 After you have validated your cycles wallet balance, you can register, build, and deploy your sample application.
 
-To deploy your first application on the {IC}:
+To deploy your first application on the {IC} blockchain:
```
Comment (andreacerulli) [inline modules/quickstart/pages/network-quickstart.adoc:346]:
```suggestion
To deploy your first application on the {IC} blockchain mainnet:
```

File: modules/quickstart/pages/network-quickstart.adoc [hunk 7]
```diff
@@ -361,12 +361,12 @@ For more information about this step, see link:../developers-guide/webpack-confi
 dfx deploy --network ic
 ----
 +
-The `+--network+` option specifies the network alias or URL for deploying the application.
-This option is required to install on the {IC} running remotely.
+The `+--network+` option specifies the network alias or URL for deploying the dapp.
+This option is required to install on the {IC} blockchain.
```
Comment (andreacerulli) [inline modules/quickstart/pages/network-quickstart.adoc:365]:
```suggestion
This option is required to install on the {IC} blockchain mainnet.
```

File: modules/quickstart/pages/network-quickstart.adoc [hunk 8]
```diff
@@ -421,7 +421,7 @@ Let's take a closer look at this example:
 +
 -- 
 
-* Using the `+--network ic+` option indicates that the canister you want to call is deployed on the `+ic+` network. The `+ic+` network alias is an internally-reserved alias for accessing the {IC}.
+* Using the `+--network ic+` option indicates that the canister you want to call is deployed on the `+ic+`. The `+ic+` network alias is an internally-reserved alias for accessing the {IC} blockchain.
```
Comment (andreacerulli) [inline modules/quickstart/pages/network-quickstart.adoc:424]:
```suggestion
* Using the `+--network ic+` option indicates that the canister you want to call is deployed on the `+ic+`. The `+ic+` network alias is an internally-reserved alias for accessing the {IC} blockchain mainnet.
```

**Review by andreacerulli — APPROVED** [review]
LGTM

File: modules/quickstart/pages/network-quickstart.adoc (hunk 2 above)
Comment (yvonneanne) [reply modules/quickstart/pages/network-quickstart.adoc:125]:
```suggestion
== Check the connection to the {platform} mainnet
```

**Pull request merged**